hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/interfaces/TestHandler.java

```diff
@@ -58,6 +60,7 @@
 
   @Before
   public void setup() throws Exception {
+    DefaultMetricsSystem.instance().shutdown();
```

Comment:
Do we still need this one after the added cleanup in TestHddsDispatcher?

Reply:
My mistake, I will remove this